core/common/src/main/java/alluxio/wire/WorkerInfo.java

```diff
@@ -190,6 +191,22 @@ public boolean equals(Object o) {
         && mStartTimeMs == that.mStartTimeMs;
   }
 
+  /**
+   * Uses the last contact seconds of to determine order from most recently contacted to least
```

Comment:
We can remove the "Uses the last contact seconds of to" part since it's an implementation detail, and start from "Determines order from ..."